content/en/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -19,30 +19,552 @@ admission controller. This can be done effectively using a combination of dry-ru
+  aspect of PSP, you will need to either modify your workloads to meet the Pod Security constraints,
+  or use a [Mutating Admission Webhook][dynamic-admission] to make those changes. See [Simplify &
+  Standardize PodSecurityPolicies](#simplify-psps) below for more detail.
+- **Fine-grained control over policy definition** - Pod Security Admission only supports [3 standard
+  levels](/docs/concepts/security/pod-security-standards/). If you require more control over
+  specific constraints, then you will need to use a [Validating Admission
+  Webhook][dynamic-admission] to enforce those policies.
+- **Sub-namespace policy granularity** - PodSecurityPolicy lets you bind different policies to
+  different Service Accounts or users, even within a single namespace. This approach has many
+  pitfalls and is not recommended, but if you know what you're doing and need this feature you will
+  need to use a 3rd party webhook instead. The exception to this is if you only need to exempt
+  specific users or [RuntimeClasses](/docs/concepts/containers/runtime-class/), in which case Pod
+  Security Admission does expose some [static configuration for
+  exemptions](/docs/concepts/security/pod-security-admission/#exemptions).
+
+Even if Pod Security Admission does not meet all of your needs it was designed to be _complementary_
+to other policy enforcement mechanisms, and can provide a useful fallback running alongside other
+admission webhooks.
+
+[dynamic-admission]: /docs/reference/access-authn-authz/extensible-admission-controllers/
+
+## 1. Review namespace permissions
+
+Pod Security Admission is controlled by [labels on
+namespaces](/docs/concepts/security/pod-security-admission/#pod-security-admission-labels-for-namespaces).
+This means that anyone who can update (or patch or create) a namespace can also modify the Pod
+Security level for that namespace, which could be used to bypass a more restrictive policy. Before
+proceeding, ensure that only trusted, privileged users have these namespace permissions. It is not
+recommended to grant these powerful permissions to users that shouldn't have elevated permissions,
+but if you must you will need to use an [admission webhook][dynamic-admission] to place additional
+restrictions on Pod Security labels.
+
+## 2. Simplify & Standardize PodSecurityPolicies {#simplify-psps}
+
+In this section, you will reduce mutating PodSecurityPolicies and remove options that are outside
+the scope of the PodSecurityStandards. You should make the changes recommended here to an offline
+copy of the original PodSecurityPolicy being modified. The cloned profile should have a different
+name that is alphabetically before the original (for example, prepend a `0` to it). Do not create the
+new policies in Kubernetes yet - that will be covered in the [Rollout the updated
+policies](#psp-update-rollout) section below.
+
+### 2.a. Eliminate purely mutating fields
```

Comment:
This heading doesn't feel quite right - do you have any other wording you considered here?

Reply:
Let's live with it. I can't suggest something better.